<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BenchmarkScreen</class>
 <widget class="QWidget" name="BenchmarkScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QWidget" name="upperContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="contextLayout" stretch="0,4">
        <item>
         <widget class="QLabel" name="contextLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Device | Platform</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="contextComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="layout_top">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="BenchmarkListWidget" name="benchmarkListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>255</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>255</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QListWidget 
{ 
/*background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(41, 41, 41, 255), stop:1 rgba(85, 85, 85, 255));  */
background: rgba(240,240,240,255);
border: 0px;
}

QListWidget::item:hover 
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150, 150, 150, 255), stop:1 rgba(200, 200, 200, 255));
}

QListWidget::item:selected 
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(200, 200, 200, 200), stop:1 rgba(220, 220, 220, 255)); 
border-top: 4px groove gray; 
color: #404040;
}
QIcon
{
background: red;selection-color: red; 
}


QIcon::selected
{
background: red;selection-color: red; 
}

QListWidget::item 
{ 
padding: 10px; 
color: red; 
border-bottom: 2px groove gray; 
border-left: 2px groove gray; 
border-right: 2px groove gray; 
border-top: 2px groove gray; 
}</string>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="autoScroll">
           <bool>false</bool>
          </property>
          <property name="autoScrollMargin">
           <number>0</number>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="flow">
           <enum>QListView::TopToBottom</enum>
          </property>
          <property name="isWrapping" stdset="0">
           <bool>false</bool>
          </property>
          <property name="viewMode">
           <enum>QListView::ListMode</enum>
          </property>
          <property name="selectionRectVisible">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>All</string>
           </property>
           <property name="background">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
           <property name="icon">
            <iconset resource="../../resources/icons.qrc">
             <normaloff>:/icons/icons/checkTrue.png</normaloff>:/icons/icons/checkTrue.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blas3</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/icons.qrc">
             <normaloff>:/icons/icons/checkTrue.png</normaloff>:/icons/icons/checkTrue.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Copy</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/icons.qrc">
             <normaloff>:/icons/icons/checkTrue.png</normaloff>:/icons/icons/checkTrue.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sparse</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/icons.qrc">
             <normaloff>:/icons/icons/checkTrue.png</normaloff>:/icons/icons/checkTrue.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vector</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/icons.qrc">
             <normaloff>:/icons/icons/checkTrue.png</normaloff>:/icons/icons/checkTrue.png</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="expert_configBoxPlaceholder" native="true"/>
        </item>
        <item>
         <widget class="QCustomPlot" name="finalResultPlot" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>255</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>255</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="precisionGroupBox">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox
{
    border: 2px solid gray;
   /* border-radius: 5px;*/
border-bottom: 2px groove gray; 
border-left: 2px groove gray; 
border-right: 2px groove gray; 
border-top: 2px groove gray; 
    /*margin-top: 1ex; /* leave space at the top for the title */
}
/*
QGroupBox::title  {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
/*
}*/

QPushButton 
{ 
background: rgba(240,240,240,255);

border-bottom: 1px groove  rgba(240,240,240,255); 
border-left: 1px groove  rgba(240,240,240,255); 
border-right: 1px groove  rgba(240,240,240,255); 
border-top: 1px groove  rgba(240,240,240,255); 
}

QPushButton:hover
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(200, 200, 200, 200), stop:1 rgba(220, 220, 220, 255)); 
border-bottom: 1px groove  rgba(240,240,240,255); 
border-left: 1px groove  rgba(240,240,240,255); 
border-right: 1px groove  rgba(240,240,240,255); 
border-top: 1px groove  rgba(240,240,240,255); 
}
QPushButton:checked 
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(200, 200, 200, 200), stop:1 rgba(220, 220, 220, 255)); 
border-bottom: 1px groove gray; 
border-left: 1px groove gray; 
border-right: 1px groove gray; 
border-top: 1px groove gray; 
}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QPushButton" name="doublePrecisionButton">
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Double</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/icons.qrc">
             <normaloff>:/icons/icons/checkTrue.png</normaloff>:/icons/icons/checkTrue.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="singlePrecisionButton">
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Single</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/icons.qrc">
             <normaloff>:/icons/icons/empty.png</normaloff>:/icons/icons/empty.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="startStopProgressLayout">
        <item>
         <widget class="QPushButton" name="startBenchmarkButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>155</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton  {
    border: 2px solid gray;
margin: 0px;
   /* border-radius: 5px;*/
border-bottom: 2px groove gray; 
border-left: 2px groove gray; 
border-right: 2px groove gray; 
border-top: 2px groove gray; 
}
QPushButton:hover {
    border: 2px solid gray;
margin: 0px;
color: green;
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 gray, stop:1 rgba(200, 200, 200, 255));
  /*  border-radius: 5px;*/


border-bottom: 2px groove gray; 
border-left: 2px groove gray; 
border-right: 2px groove gray; 
border-top: 2px groove gray; 
}

QPushButton:pressed 
{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(200, 200, 200, 200), stop:1 rgba(220, 220, 220, 255)); 
border-top: 2px groove gray; 
color: #404040;
}</string>
          </property>
          <property name="text">
           <string>START</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources/icons.qrc">
            <normaloff>:/icons/icons/start.png</normaloff>:/icons/icons/start.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopBenchmarkButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>155</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton  {
    border: 2px solid gray;
margin: 0px;
   /* border-radius: 5px;*/
border-bottom: 2px groove gray; 
border-left: 2px groove gray; 
border-right: 2px groove gray; 
border-top: 2px groove gray; 
}
QPushButton:hover {
    border: 2px solid gray;
margin: 0px;
color: green;
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 gray, stop:1 rgba(200, 200, 200, 255));
  /*  border-radius: 5px;*/


border-bottom: 2px groove gray; 
border-left: 2px groove gray; 
border-right: 2px groove gray; 
border-top: 2px groove gray; 
}

QPushButton:pressed 
{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(200, 200, 200, 200), stop:1 rgba(220, 220, 220, 255)); 
border-top: 2px groove gray; 
color: #404040;
}</string>
          </property>
          <property name="text">
           <string>STOP</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources/icons.qrc">
            <normaloff>:/icons/icons/stop.png</normaloff>:/icons/icons/stop.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QProgressBar  {
    border: 2px solid gray;
margin: 0px;
   /* border-radius: 5px;*/
border-bottom: 2px groove gray; 
border-left: 2px groove gray; 
border-right: 2px groove gray; 
border-top: 2px groove gray; 
}
QPushButton:hover {
    border: 2px solid gray;
margin: 0px;
color: green;
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 gray, stop:1 rgba(200, 200, 200, 255));
   /* border-radius: 5px;*/
border-bottom: 2px groove gray; 
border-left: 2px groove gray; 
border-right: 2px groove gray; 
border-top: 2px groove gray; 
}

QPushButton:pressed 
{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(200, 200, 200, 200), stop:1 rgba(220, 220, 220, 255)); 
border-top: 2px groove gray; 
color: #404040;
}</string>
          </property>
          <property name="maximum">
           <number>26</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="layoutToggle">
     <item>
      <widget class="QPushButton" name="toggleFullscreenButton">
       <property name="text">
        <string>Toggle Fullscreen</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/icons.qrc">
         <normaloff>:/icons/icons/upArrow.png</normaloff>:/icons/icons/upArrow.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTabWidget" name="fullscreenPlotsWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>src/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BenchmarkListWidget</class>
   <extends>QListWidget</extends>
   <header>src/ui/widgets/benchmarklistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
